en/attributes.html

```diff
@@ -123,10 +141,19 @@ <h2 id="data-structures">Data structures</h2>
       One large memory buffer holding references (32 bit) to a memory structure
       with objects per unique value.
     </td>
+  </tr><tr>
+    <td>Tensor</td>
+    <td>Fixed / Variable</td>
+    <td>
+      Like the "D" attribute - a tensor is a multivalued datatype with sparse and/or dense dimensions,
```

Comment:
We should use the terms "mapped and indexed dimensions" to be aligned with https://docs.vespa.ai/en/reference/tensor.html. I am also not sure we should say its a multivalue datatype.

Reply:
"mapped and indexed dimensions" better, yes. removed multivalue

Reply:
not sure what we put in multivalue semantics, though, as a tensor has multiple values, but nevermind